Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -16,6 +17,9 @@ namespace Microsoft.MixedReality.Toolkit.SDK.UX.Cursors
     /// </summary>
     public class BaseCursor : MonoBehaviour, IMixedRealityCursor
     {
+        private static IMixedRealityInputSystem inputSystem = null;
```

Comment:
Why not use one of the new base classes?

Reply:
This -is_ the base class 😅

Reply:
lol, but you know what I mean.  Like InputSystemGlobalListener, which the exact same code as used here.